src/tests/go.mod

```diff
@@ -1,39 +1,17 @@
 module tests
 
 require (
-	contrib.go.opencensus.io/exporter/ocagent v0.4.3 // indirect
```

Comment:
Why are so many importing modules removed?

Reply:
@YanzhaoLi This is done by `go mod tidy` and `go mod vendor` actions automatically. Our tests have passed, so I think these removed packages are not needed at all, now!